launcher/internal/signaturediscovery/client.go

```diff
@@ -80,13 +80,17 @@ func (c *Client) FetchImageSignatures(ctx context.Context, targetRepository stri
 func (c *Client) pullTargetImage(ctx context.Context, targetRepository string) (containerd.Image, error) {
 	targetImageRef := fmt.Sprint(targetRepository, ":", formatSigTag(c.OriginalImageDesc))
 
+	// Pull signature image from a public repository.
+	if c.remoteResolverFetcher == nil {
+		return c.imageFetcher(ctx, targetImageRef)
+	}
+
 	// Refresh resolver before pulling container image.
-	if resolver, err := c.remoteResolverFetcher(ctx); err == nil {
```

Comment:
as a small nit I'd like to rename "remoteResolverFetcher" to perhaps "refreshResolver"

It looks like in the way we use it there is only "refreshing" happening. It is a function after all, not an object. So I like having a verb in it.

Anyway, its a nit, but this was a bit hard to read.

If you're down, feel free to close these and open in another PR

Reply:
Done.